bin/update-scripts/stage/update-stage-product-details.sh

```diff
@@ -0,0 +1,12 @@
+#!/bin/bash
+
+DEPLOY_SCRIPT=/data/bedrock-stage/deploy
+WORKING_DIR=/data/bedrock-stage/src/www.allizom.org-django/bedrock
+SITE_NAME=www.allizom.org
+PYTHON=../venv/bin/python
+PD_PATH=lib/product_details_json
+
+cd $WORKING_DIR
+$PYTHON manage.py update_product_details
+$DEPLOY_SCRIPT -q "${SITE_NAME}-django/bedrock/$PD_PATH"
```

Comment:
I agree, but I think a lot of these scripts could use some refactoring for such a thing, so I'd rather do that in a separate PR in order to get them all on the same page. That large a refactor doesn't belong in this PR IMHO.